src/components/PopupBase/__tests__/__snapshots__/PopupBase.spec.tsx.snap

```diff
@@ -1,175 +1,67 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<PopupBase> that adds basic anchor/popup functionality to rendered components rendering should render popup in portal when requested 1`] = `
-<PopupBase
-  anchorRenderer={[Function]}
-  popupRenderer={[Function]}
-  renderInPortal={true}
->
-  <Button
-    onClick={[Function]}
+<div>
```

Comment:
I meant this snapshot, probably mentioned in the wrong test.

Reply:
:+1: